### DISCUSSION TITLE
Save and Import or Open

### DISCUSSION THREAD

**Discussion by mwarner485** [vishnuraghavb/EnBizCard#41] [Ideas]
Any roadmap item for being able to save the card profile when creating so that if I need to make major changes, I can just open or import it into the web builder to make changes rather than starting from scratch again?

**Comment by ALXRITR** [comment]
Yeah would be soo awesome!

**Comment by benjezza** [comment]
Even if its an export and load json file?  would be very helpful when creating cards for a team!
